docker/docker-compose.yml

```diff
@@ -17,8 +17,29 @@ x-worker: &base-worker
 version: "2.4"
 services:
 
+  traefik:
+    image: traefik:v2.4
+    container_name: traefik
+    command: >
+      --api.insecure=true
+      --serversTransport.insecureSkipVerify=true
```

Comment:
if we eventually remove nginx  guess we will want this disabled for prod?

Reply:
This is a slightly misleading argument.  It allows inscure SSL behind the proxy, not in front.  So like if girder were sending self-signed certs.  It can also be removed